### ISSUE TITLE
Verify license and import park data from SanGIS

### ISSUE THREAD

**Issue by cgiovando** [opensandiego/sd-playgrounds-parks#2]
Use license of SanGIS' "PARKS_ACTIVE_USE" layer as reported [here](http://rdw.sandag.org/file_store/Park/Parks_Active_Use.pdf) references SANDAG and Rand McNally copyright:

> Note: This product may contain information from the SANDAG Regional
> Information System which cannot be reproduced without the written permission
> of SANDAG. This product may contain information reproduced with permission
> granted by Rand McNally & Company® to SanGIS. This map is copyrighted by
> Rand McNally & Company®. It is unlawful to copy or reproduce all or any part
> thereof, whether for personal use or resale, without the prior, written permission
> of Rand McNally & Company®.

License should be verified with SanGIS to make sure it can be used within the app.

Data layers in shapefile polygon format can be downloaded from SanGIS:
- [Active use park areas in San Diego County](http://rdw.sandag.org/file_store/Park/Parks_Active_Use.zip)
- [City of San Diego Park and Recreation Department parks](http://rdw.sandag.org/file_store/Park/Parks_SD.zip)
- [County of San Diego Park and Recreation Department parks](http://rdw.sandag.org/file_store/Park/Parks_CN.zip)

**Comment by ghost** [comment]
I am working to verify this, I have a GIS Supervisor here in our Department who works with SanGIS and hopefully should be able to provide us with the answer.

**Comment by ghost** [comment]
From my SanGIS engineer on staff:

Hi Gary,

You have to accept the Use Agreement to download the data.  Once submitted, you're good to go.  The user couldn't have obtained the files otherwise.  As long as you're not modifying the polygons and presenting the new data as SANDAG or SanGIS products, you're okay.

Thanks,
scott

**Comment by cgiovando** [comment]
Thanks @Shaynei 

Just to clarify: there is no explicit mention of a data license in the SanGIS use agreement presented when entering the Regional Data Warehouse. The Rand McNally reference is still there with a SanGIS "All Rights Reserved" statement.

From what I understand [CC-BY](http://creativecommons.org/licenses/by/4.0/) is probably the closest license, but would be nice to confirm that it is at least compatible with [ODBL](http://opendatacommons.org/licenses/odbl/summary/) since we are planning to import that data into OpenStreetMap.

CCing @MrMaksimize who may have experience with other open data licenses used within the City and County.

**Comment by jj0hns0n** [comment]
Can we get SANGIS to sign something saying its ok to import to OSM? There
is some precedent for this. http://wiki.openstreetmap.org/wiki/SanGIS

**Comment by jj0hns0n** [comment]
We also use this letter on page 87 of this guide for other projects I work
on.

http://www.opencitiesproject.org/OpenCities_Book_LoRes.pdf

**Comment by ghost** [comment]
Spoke with SanGIS and this is their reply:

Gary,

Okay, I think I see where the problem is now.  I believe SanGIS needs to change their legal notice to remove the reference to Rand McNally.  That reference is an appropriate notation on a map that uses the Rand McNally data (which the City and County can) but it is not data that is on the public data warehouse and therefore should not apply.  I will work on getting it changed.

Scott’s initial reply is correct:  If they download data from our public data warehouse they can use it anyway they want.  If they use it unaltered we (SanGIS) ask that they provide a credit to SanGIS.  But that is not a requirement and we can’t compel them to do that.  If they use the data in any other way – add to, subtract from, “enhance”, etc. – then we specifically require them not to credit SanGIS.

The bottom line is:  if they got it from our public data warehouse (www.sangis.org) then they can use it freely, including loading it into Open Street Map.  In fact, I think a lot of the SanGIS data has already been loaded there.

I’ll work on changing the legal notice so that it doesn’t refer to the Rand McNally copyright.

Brad Lind
